### PR TITLE
[MODULAR] Fix typo in anesthetic machine deconstruction

### PR DESCRIPTION
**Pull request by out-of-phaze**
## About The Pull Request
Fixes a kinda incomprehensible message and a typo in anesthetic machine deconstruction.

## How This Contributes To The Skyrat Roleplay Experience
Cleaning up the code in the modular Skyrat folder.

## Proof of Testing
Purely a text change.

### CODE REVIEW

File: modular_skyrat/modules/medical/code/anesthetic_machine.dm [hunk 1]
```diff
@@ -39,7 +39,7 @@
 		return TRUE
 
 	if(attached_tank)
-		to_chat(user, span_warning("[attached_tank] inside of the [src] deconsturction."))
+		to_chat(user, span_warning("\The [attached_tank] must be removed from \the [src] first!"))
```
Comment (Zonespace27) [inline modular_skyrat/modules/medical/code/anesthetic_machine.dm]: adding \the does nothing, remove them
image

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/medical/code/anesthetic_machine.dm (hunk 1 above)
Comment (out-of-phaze) [reply modular_skyrat/modules/medical/code/anesthetic_machine.dm]: Didn't you just suggest changes on another PR to add that...? I mean, I know it doesn't do anything, but I find that using `\the` and `\The` is easier to read and more explicit. I can change it, though.

Comment (Zonespace27) [reply modular_skyrat/modules/medical/code/anesthetic_machine.dm]: P sure i suggested a removal

Comment (Zonespace27) [inline modular_skyrat/modules/medical/code/anesthetic_machine.dm]:
```suggestion
		to_chat(user, span_warning("[attached_tank] must be removed from [src] first!"))
```

**Review by Zonespace27 — APPROVED** [review]

**Pull request merged**